proposals/lip-0047.md

```diff
@@ -572,22 +572,7 @@ validatorSignatureMessageSchema = {
 
 #### shuffleValidatorsList
 
-The function reorders and returns the list of validators as specified in [LIP 0003][lip-0003#specs].
-
-##### Execution
-
-```python
-def shuffleValidatorsList(validatorsAddresses: list[Address], randomSeed: RandomSeed) -> list[Address]:
-    roundHash = {}
-    for address in validatorsAddresses:
-        roundHash[address] = SHA-256(randomSeed + address)
-
-    # Reorder the validator list.
-    shuffledValidatorAddresses = sort validatorsAddresses where address1 < address2 if (roundHash[address1] < roundHash[address2])
-                                 or ((roundHash[address1] == roundHash[address2]) and address1 < address2)         
-
-    return shuffledValidatorAddresses
-```
+The function reorders and returns the list of validators as specified in [LIP 0003][lip-0003#specs]. It must be implemented exactly as the `shuffleValidatorsList` function in the [PoS module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#shufflevalidatorslist).
```

Comment:
```suggestion
The function reorders and returns the list of validators. It must be implemented exactly as the `shuffleValidatorsList` function in the [PoS module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#shufflevalidatorslist).
```

Reply:
I think the historic reference to LIP 0003 is not important here and rather confusing (there should be one source of truth for the specifications).